benchmark/tests/test_sample_genes.py

```diff
@@ -0,0 +1,63 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+from shutil import rmtree
+from tempfile import mkdtemp
+from os.path import join, dirname, realpath
+from click.testing import CliRunner
+
+from benchmark.sample_genes import (sample_genes,
+                                    _main)
+
+
+class SampleGenesTests(TestCase):
+    """ Test for sample_taxa.py """
+
+    def setUp(self):
+        """ Set up working directory and test files
+        """
+        # test output can be written to this directory
+        self.working_dir = mkdtemp()
+
+        # test data directory
+        datadir = join(dirname(realpath(__file__)), 'data', 'simplified')
```

Comment:
This is just for testing purposes.  Personally, I use it often for my own solutions since it is pretty easy / readable.
Since skbio is already a dependency, using it here is not an issue.  But I personally don't have preferences wrt this.